ext/openssl/tests/gh19369.phpt

```diff
@@ -0,0 +1,24 @@
+--TEST--
+GH-19369: openssl_sign with alias algorithms
+--EXTENSIONS--
+openssl
+--SKIPIF--
+<?php
+if (!in_array('sha256WithRSAEncryption', openssl_get_md_methods(true))) {
+    die('skip sha256WithRSAEncryption alias not present');
+}
+?>
+--FILE--
+<?php
+$digests             = openssl_get_md_methods();
+$digests_and_aliases = openssl_get_md_methods(true);
+$digest_aliases      = array_diff($digests_and_aliases, $digests);
```

Comment:
I believe this code is redundant if a hard-coded alias is used in the test.

Reply:
Ah yeah I initially tried the all alias test as suggested here but that cannot work because not all aliases can be used for sign digest.

Anyway fixed in https://github.com/php/php-src/commit/066a977840f9993d375c18bcd043cf0c2920bb09